src/fare-contracts/utils.ts

```diff
@@ -172,7 +172,11 @@ export const useTariffZoneSummary = (
   );
   const zoneSelectionModeDisabledForProduct =
     fareProductTypeConfig?.configuration.zoneSelectionMode === 'none';
-  if (zoneSelectionModeDisabledForProduct) return undefined;
+
+  const usesZoneOfferEndpointForProduct =
+    fareProductTypeConfig?.configuration.offerEndpoint === 'zones';
```

Comment:
I don't think this is correct. If there is zone-selection-mode for the product, then it should probably have a tariff zone summary.

Reply:
Updated to check if the offer endpoint is authority instead

Reply:
The change we do now is to decouple what endpoint to use from the zone-selection-mode. Thus the tariff-zone summary shouldn't be affected by what endpoint is configured. 

What is the origin of this change? The hjem-jobb-hjem will have zone-selection-mode none, and will return `undefined` as tariff-zone summary even without this change?

Reply:
It is to display the zone in the fare contract item : https://www.figma.com/file/zdZwvobgpEWSagKt0tderx/App?type=design&node-id=24257-23134&mode=design&t=4VLBkHRJxSL2KJRF-4

Since the zone selection is `none`, the `zone` part is not shown in the farecontract, that's why I added another check, because this hjemjobbhjem is a bit weird, the zone selection is none, but it is only valid for zone A, therefore we need to show the zone on the ticket.

<img width="371" alt="image" src="https://github.com/AtB-AS/mittatb-app/assets/1777333/e13bd943-1401-4d9c-b044-1b17df51e9f7">

Reply:
Aight. We should find another way to handle it, so there isn't any coupling between zone selection mode/zone presentation, and what offer endpoint is used. Let me take a look.